frontend/i18next.scanner/en/translation.json

```
@@ -244,6 +244,7 @@
   "User administration": "User administration",
   "Username can't contain any whitespace": "Username can't contain any whitespace",
   "Username must be at least {{minimumCharactersUsername}} characters": "Username must be at least {{minimumCharactersUsername}} characters",
+  "Username must have between {{minimumCharactersUsername}} and {{maximumCharactersUsername}} characters": "Username must have between {{minimumCharactersUsername}} and {{maximumCharactersUsername}} characters",
```

Would this be better?

```suggestion
  "Username must be between {{minimumCharactersUsername}} and {{maximumCharactersUsername}} characters long": "Username must be between {{minimumCharactersUsername}} and {{maximumCharactersUsername}} characters long",
```